types/mapcmd/map.go

```diff
@@ -109,17 +109,22 @@ func decorateCommandWithMapNameFlags(cmd *cobra.Command, mapName *string) {
 	cmd.MarkFlagRequired("name")
 }
 
-func decorateCommandWithKeyFlags(cmd *cobra.Command, mapKey *string) {
-	cmd.Flags().StringVarP(mapKey, "key", "k", "", "key of the map")
+func decorateCommandWithKeyFlags(cmd *cobra.Command, mapKey, mapKeyType *string) {
+	flags := cmd.Flags()
+	flags.StringVarP(mapKey, "key", "k", "", "key of the map")
+	flags.StringVarP(mapKeyType, "key-type", "", "string", "type of the key, one of: string, json, int(8,16,32,64), float(32,64)")
```

Comment:
Maybe it is better to list all of them like int8, int16 etc. What do you think? But regardless of it, looks like it does not fit to the prompt: 
![image](https://user-images.githubusercontent.com/32355782/171357108-22cee0e1-c1aa-4ffe-be2b-9446512ec0b3.png)

Can we make it fit via displaying it in multiple lines etc?